Parser/Rule.php

```diff
@@ -29,7 +29,7 @@
  *
  * @package Kadet\Highlighter\Parser
  *
- * @property Language              $language
+ * @property Language|false|null   $language
```

Comment:
I'm not sure if `null` is really a supported value here. The default value for the option is `false` and if `null` is not supported then we have to fix the condition in https://github.com/kadet1090/KeyLighter/blob/18d39a071b347328f326dbc95bba2a2ae6cac750/Parser/Token/LanguageToken.php#L55 to compare against `false` instead (or use `empty()`)